_layouts/redirect.html.old

```diff
@@ -1,3 +1,6 @@
+<!--
```

Comment:
What is this file doing? Can we remove it?

Reply:
If I left it, the jekyll redirection didnt' work, and `/developersmeetup/` redirected to `docs.julialang.org/en/latest/developersmeetup`